lib/vitalsigns/device_lookup.rb

```diff
@@ -0,0 +1,18 @@
+module Vitalsigns
+  # Find device/fileSystem path
+  class DeviceLookup
+    class << self
+      def default
+        find_by_dir
+      end
+
+      def find_by_dir(dire_path = '/var')
+        cmd = "df -h #{dire_path} | awk '{ print $1}'"
```

Comment:
This would avoid string interpolation done on line #12 to 14.
`cmd = "df -h #{dire_path} | sed -n '2p' | awk '{ print $1}'"`